frontend/packages/css/colors.css

```diff
@@ -1,5 +1,3 @@
-@import 'primitiveColors.css';
```

Comment:
let's not remove @import 'primitiveColors.css'; in scope of this pr. This might cause unexpected regressions